src/NServiceBus.Core/Transports/Learning/DirectoryBasedTransaction.cs

```diff
@@ -86,6 +134,9 @@ public void Complete()
         List<OutgoingFile> outgoingFiles = new List<OutgoingFile>();
         string transactionDir;
 
+        const string CommittedDirName = ".committed";
```

Comment:
can we use this constant here too: https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Transports/Learning/LearningTransportMessagePump.cs#L29 ?